systems/data-plan/package/pkg/server/package.go

```diff
@@ -113,23 +127,29 @@ func (p *PackageServer) Update(ctx context.Context, req *pb.UpdatePackageRequest
 	logrus.Infof("Update Package Uuid: %v, Name: %v, SimType: %v, Active: %v, Duration: %v, SmsVolume: %v, DataVolume: %v, Voice_volume: %v",
 		req.Uuid, req.Name, req.SimType, req.Active, req.Duration, req.SmsVolume, req.DataVolume, req.VoiceVolume)
 	_package := db.Package{
-		Name:         req.GetName(),
-		Sim_type:     req.GetSimType().String(),
-		Active:       req.Active,
-		Duration:     uint(req.GetDuration()),
-		Sms_volume:   uint(req.GetSmsVolume()),
-		Data_volume:  uint(req.GetDataVolume()),
-		Voice_volume: uint(req.GetVoiceVolume()),
-		Org_rates_id: uint(req.GetOrgRatesId()),
+		Name:        req.GetName(),
+		SimType:     1,
```

Comment:
Using the type enum (instead of the number) will make it easier to update/debug in the future

Reply:
addressed